apps/autoscheduler/weights/head1_conv1_bias.data

```diff
@@ -0,0 +1 @@
+������a�[�M��|�ɿ`�H?�>Ib?
```

Comment:
I forget: is using a binary-float format really a substantial win over a human-readable format? (I doubt time-to-read the file is an issue, so are we concerned about bit accuracy?)

Reply:
This seems simpler - in the case where they're not embedded into the compiler, it means you can just fread them, instead of needing to parse ascii floats.